include/k4a/k4a.h

```diff
@@ -33,6 +33,21 @@ extern "C" {
  * @{
  */
 
+
+/** get device sync mode
+ *
+ * \remarks This API is currently mainly used to get device sync mode
```

Comment:
Here you can add comments to explain that the device synchronization mode will change according to the mode configured in the start_cameras function.

Reply:
ok